collections/includes_value.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Returns whether the given value is part of the collection.
+ *
+ * Example:
+ * ```ts
+ * import { includesValue } from "./includes_value.ts";
+ * import { assert } from "../testing/asserts.ts";
+ *
+ * const input = {
+ *   first: 33,
+ *   second: 34,
+ * };
+ *
+ * assert(includesValue(input, 34));
+ */
+
+export function includesValue<T>(
+  collection: Record<string, T>,
```

Comment:
We should rename the argument to `record` or something more specific. Also marking as `readonly` here is good practice to signal that no mutation will happen.

Reply:
I don't believe we can do `readonly` for `Record`s, getting this:
`'readonly' type modifier is only permitted on array and tuple literal types.`

Reply:
You could do `Readonly<Record<>>`

Reply:
The typings seem to work out using `Readonly<Record<string, T>>`, I will push that.